scripts/sync_tx_src.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+/**
+ * @fileoverview
+ * Script to upload a source en.json file to a particular transifex project resource.
+ * Expects that the project and resource have already been defined in Transifex, and that
+ * the person running the script has the the TX_TOKEN environment variable set to an api
+ * token that has developer access.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const transifex = require('transifex');
+
+const args = process.argv.slice(2);
+
+const usage = `
+Sync English source strings with Transifex. Usage:
```

Comment:
One thought I'm having reading these is that "sync" is used both for pushing/uploading and for pulling/downloading. It might be clearer if we used distinct words in the filenames and comments, like "push" and "pull", or "upload sync" and "download sync", or "sync_to" and "sync_from".

Reply:
Good point - I'll rename these push and pull.